### DISCUSSION TITLE
Save all and close all buffers without quitting

### DISCUSSION THREAD

**Discussion by kaleidawave** [helix-editor/helix#5011] [Ideas]
Looking for a command that is `write-quit-all` without quitting helix. or a combination of `write-all` and `buffer-close-all` (`buffer-close-all` if changes haven't been saved). Does this exist?

**Comment by the-mikedavis** [comment]
There isn't something that combines these two - it would take two separate commands to write and then close all buffers.

**Reply by kaleidawave** [reply]
Ah okay. It's not possible to add new commands via a plugin of sorts right now, right? May create a feature request issue then

**Reply by the-mikedavis** [reply]
You could bind a key to execute both at once

```toml
[keys.normal]
space.q = [":write-all", ":buffer-close-all"]
```

Creating a new typable command (ones starting with `:`) is not yet possible and there is an issue for that: https://github.com/helix-editor/helix/issues/4423
